modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request_expiration.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyRequestExpiration < ApplicationRecord
+    has_one :power_of_attorney_request_resolution,
+            as: :resolving,
+            inverse_of: :resolving,
+            dependent: :destroy
+
+    # Validations
+    validates :id, presence: true
```

Comment:
`validates :id, presence: true` is something i've never seen before. I think it shouldn't be there.